docs/src/main/asciidoc/spring-web-guide.adoc

```diff
@@ -103,23 +64,16 @@ import org.springframework.web.bind.annotation.PathVariable;
 @RequestMapping("/greeting")
 public class GreetingController {
 
-    private final GreetingBean greetingBean;
-
-    public GreetingController(GreetingBean greetingBean) {
-        this.greetingBean = greetingBean;
+    @GetMapping
+    public String hello() {
+        return "hello";
     }
-
-    @GetMapping("/{name}")
-    public Greeting hello(@PathVariable(name = "name") String name) {
-        return new Greeting(greetingBean.greet(name));
-    }
-}
+}1
```

Comment:
Is `1` here a typo?

Reply:
I'll fix it

Reply:
Looking at: https://raw.githubusercontent.com/quarkusio/quarkus/1c58752dd3424b52113bbfba8cc2ffd6665d305f/docs/src/main/asciidoc/spring-web-guide.adoc it seems like ``` is still there